src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -158,7 +163,7 @@ private void WriteFile()
                 case TargetLanguage.VB:
                     WriteLine("' <auto-generated />"); break;
                 default:
-                    throw new ArgumentException("Unexpected target language", nameof(_targetLang));
+                    throw new InvalidOperationException($"Unexpected target language: '{_targetLang}'");
```

Comment:
I'm not sure if `ExceptionUtilities.Unreachable` is more suitable?

Reply:
Thank you, I will make use of that.

Reply:
If `ExceptionUtilities.UnexpectedValue` is available here that might be good to use.

Reply:
if it isn't available, is it okay to add a `using`?

Reply:
If just adding `using Roslyn.Utilities;` makes it available, then it's fine to use it.